addons/hr_recruitment/views/res_config_settings_views.xml

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+        <record id="open_job_board_modules" model="ir.actions.act_window">
+            <field name="name">Job Boards</field>
+            <field name="res_model">ir.module.module</field>
+            <field name="view_mode">kanban,tree,form</field>
+            <field name="context">{}</field>
+            <field name="domain">[
+                ('name', 'in', [
+                        'hr_recruitment_integration_monster',
```

Comment:
Don't hardcode module names, because you would need to update hr_recruitment to see new modules, use a domain like this ``('name', '=like', 'hr_recruitment_integration_%')``

Reply:
yup but the thing is that I need to not display the modules that are not exactly in the job boards (eg integration_website and monster_skills and _base shouldn't be included in that)